js/classes/sub-set.js

```diff
@@ -0,0 +1,99 @@
+// Variable containing availablity and exchange specific pair identifier for
+//    all exchanges
+// TODO: Begin filling this variable out
+var info = {
+  poloniex : {
+    BTCUSD : "USDT_BTC",
+    ETHBTC : "BTC_ETH"
+  },
+  gdax : {
+    BTCUSD : "BTC-USD",
+    ETHBTC : "ETH-BTC"
+  },
+  kraken : {
+    BTCUSD : "XXBTZUSD",
+    ETHBTC : "XETHXXBT"
+  },
+  bitfinex : {
+    BTCUSD : "BTCUSD",
+    ETHBTC : "ETHBTC"
+  }
+}
+
+// Function which takes in a orderbook-exchange pair, and opens a feed of data
+//    to that orderbook-exchange pair
+let open = function open(orderbook, exchange, pair){
+  var exchangeOrderbook = require("./../data-parsing/" + exchange + ".js")(orderbook);
+  exchangeOrderbook.openFeed(pair);
+}
+
+// Orderbook constructor
+// TODO: Change maps to javascript objects
+let orderbook = function orderbook(pair) {
+  return {
+    pair : pair,
```

Comment:
Additionally, highbids and low asks don't have pairs, the order book has a pair. Its required to store the actual exchange specific pair in the order book, and have a standardized version used as the key value. To access the pair of something you would need
```javascript
Exchs.poloniex.BTCUSD.pair == "USDT_BTC";
Exchs.poloniex.BTCUSD.highbids == Map[Object];
```